projects/experimental/directives/button-vertical/button-vertical.style.less

```diff
@@ -0,0 +1,32 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButton][tuiButtonVertical] {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    padding: 0.75rem;
+    grid-gap: 0.5rem;
+    gap: 0.5rem;
+    line-height: 1rem;
+    min-width: 5rem;
+    text-indent: unset;
+    font-size: var(--tui-font-text-s-2);
```

Comment:
It's entire `font`, not just `font-size`, also it would override `line-height` from line 10.